app/serializers/rest/relationship_severance_event_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class REST::RelationshipSeveranceEventSerializer < ActiveModel::Serializer
+  attributes :id, :type, :purged, :target_name, :created_at
+
+  attribute :relationships_count, if: -> { current_user.present? }
+
+  def id
+    object.id.to_s
+  end
+
+  def relationships_count
+    object.severed_relationships.where(local_account: current_user.account).count
```

Comment:
Potential performance bottleneck. Perhaps a counter column?

Reply:
We'd need yet another table, as a relationship severance event is not about a specific account, unlike this count.